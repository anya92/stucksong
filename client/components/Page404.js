import React from 'react';

import { Page404 } from '../styles/error';

export default () => (
  <Page404>
    <div>404</div>
    <h1>Page not found</h1>
  </Page404>
);
