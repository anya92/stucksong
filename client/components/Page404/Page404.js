import React from 'react';

import { Page404 } from '../../styles/info';

export default () => (
  <Page404>
    <div>404</div>
    <h1>Page not found</h1>
  </Page404>
);
