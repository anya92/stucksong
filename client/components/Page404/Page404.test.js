import React from 'react';
import { shallow } from 'enzyme';

import Page404 from './Page404';

describe('Page404 component', () => {
  it('renders Page404 component correctly', () => {
    const wrapper = shallow(<Page404 />);
    expect(wrapper).toMatchSnapshot();
  });
});
