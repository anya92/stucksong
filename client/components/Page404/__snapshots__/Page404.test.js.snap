// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page404 component renders Page404 component correctly 1`] = `
<glamorous(div)>
  <div>
    404
  </div>
  <h1>
    Page not found
  </h1>
</glamorous(div)>
`;
